CAMP.md

```diff
@@ -11,40 +11,49 @@ No Authorization (Public method)
 Creates a new user. The new user is associated with the host's domain by default. This only changes if the endpoint described in the ([CAPP](/CAPP.md)) sub-protocol is used. The user<->domain relationship is one-to-one, although one server can have users with the same name as long as they are on different domains.
 
 Request:
-```
-POST /domains/{domain}/users HTTP/1.1
 
+```javascript
+HTTP/1.1

```

Comment:
That's not how an HTTP header looks like.  
It must be:  
```
POST /domains/{domain}/users HTTP/1.1
Content-Type: application/json
```
Also, there is always a blank line after the header and then the body follows.